control/fixtures/fixtures.go

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
If this is only used in tests, should we add tags to it so that it's only built when tests are being built?